srcpkgs/modsecurity/template

```diff
@@ -15,12 +15,16 @@ license="Apache-2.0"
 homepage="https://www.modsecurity.org"
 distfiles="https://github.com/SpiderLabs/ModSecurity/releases/download/v${version}/modsecurity-v${version}.tar.gz"
 checksum=b4231177dd80b4e076b228e57d498670113b69d445bab86db25f65346c24db22
-nocross="configure: error: cannot check for file existence when cross compiling"
 
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" $makedepends"
 fi
 
+pre_configure() {
+	# Use target paths or will link against host libs
+	vsed -e "s,\${path},${XBPS_CROSS_BASE}/usr,g"	-i configure
+}
+
 pre_build() {
 	sed -ri 's|(hardcode_into_libs)=.*|\1=no|' libtool
```

Comment:
please vsed.

Reply:
Done, usually I notice and fix those lines :)